dkist/net/tests/test_helpers.py

```diff
@@ -44,6 +44,17 @@ def test_download_default_keywords(orchestrate_transfer_mock, keywords):
     )
 
 
+def test_transfer_unavailable_data(mocker):
+    get_inv_mock = mocker.patch(
+        "dkist.net.client.DKISTClient.search",
```

Comment:
Is this mocker actually being hit? I would have thought this should be

```suggestion
        "dkist.net.helpers.DKISTClient.search",
```

Reply:
If it wasn't, surely the test would hit the "An attempt was made to connect to the internet" error (which is what happens if I take the mocker out entirely).